### ISSUE TITLE
Is it possible to add aircraft radar to a second screen with this software?

### ISSUE THREAD

**Issue by NissanSUTUTU** [wysiwyng/wthud2#7]
Hello, I'm a War Thunder simulator enthusiast, and I would like to know if it is possible to add modern aircraft radar to a second monitor with this software. :)
![image](https://github.com/wysiwyng/wthud2/assets/127377346/444d7522-d7ab-4d05-85af-9e5df324dc15)

**Comment by wysiwyng** [comment]
No, unfortunately no radar data whatsoever is exposed by the interface used by this tool.
